### PR TITLE
refactor: update passport branding

### PR DESCRIPTION
**Pull request by aahna-ashina**
#4

- Replace `strip.png`
- Move Issue Date from `secondaryFields` to `headerFields`
- Rename from "Holder" to "Citizen Name"
- Rename from "Passport ID" to "Number"

Screen Shot 2022-06-28 at 5 53 59 PM

# iOS

Screen Shot 2022-06-28 at 5 49 31 PM

# Android

### CODE REVIEW

**Comment by xPi2** [comment]
28eb74fb-d4fe-47fc-b732-95251a14ee2f
This is how it looks on an iPhone 11. I think we have make some tweaks to de layout.

**Comment by xPi2** [comment]
@aahna-ashina Did you try to use something like this example for the date formatting?
```
            {
                "dateStyle" : "PKDateStyleMedium",
                "isRelative" : true,
                "key" : "doors-open",
                "label" : "Doors open",
                "timeStyle" : "PKDateStyleShort",
                "value" : "2013-08-10T19:30-06:00"
            }
 ```
 From "Fields Support Formatting" on [Pass design and creation](https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/PassKit_PG/Creating.html#//apple_ref/doc/uid/TP40012195-CH4-SW1)

**Comment by xPi2** [comment]
Finally, do you think we can fill the backFields with the passport information?
Something like:
- Issue Date
- Number
- Passport Holder
- Issuer (Nation3)

File: server/template-versions/apple/1/pass.json [hunk 1]
```diff
@@ -10,28 +10,28 @@
...
-        "label": "Passport ID"
+        "label": "Number"
       }
     ],
     "primaryFields": [],
     "secondaryFields": [
       {
         "key": "holder",
-        "label": "Holder",
+        "label": "Citizen Name",
```
Comment (xPi2) [inline server/template-versions/apple/1/pass.json:28]:
```suggestion
        "label": "Passport Holder",
```
This was my fault, the design on figma was outdated, on the passport token we are using this label.

**Comment by luisivan** [comment]
Seeing how the date looks on the top right corner (quite compressed) maybe we should move it as a normal field below the passport holder field. Thoughts @aahna-ashina?

**Comment by aahna-ashina** [comment]
> Seeing how the date looks on the top right corner (quite compressed) maybe we should move it as a normal field below the passport holder field. Thoughts @aahna-ashina?

@luisivan Yes, I think that looks better.  It would also solve the problem of the header fields being displayed in a different order on certain iOS versions and certain Android apps.

Fixed here: https://github.com/nation3/mobile-passport/pull/32/commits/a4f4ce7743b3778a120a4343214af78cdc85b001

Screen Shot 2022-06-29 at 1 07 54 PM

**Comment by aahna-ashina** [comment]
> Did you try to use something like this example for the date formatting?

@xPi2 Yes, we can change the issue date formatting to "08/06/22" as presented in the [mockup](https://github.com/nation3/brand/blob/main/passport/pass/mockup.png), but wouldn't that cause confusion?  How will you know what is the day, month and year?  Maybe you had another date formatting than "08/06/22" in mind?

**Comment by aahna-ashina** [comment]
> Finally, do you think we can fill the backFields with the passport information?

@xPi2 Done:  https://github.com/nation3/mobile-passport/pull/32/commits/d28974fd298862f6ac2f5b370e4b58e1ef72a203

**Review by xPi2 — APPROVED** [review]
I've tested the new version on iOS. Looks great!

**Review by luisivan — CHANGES_REQUESTED** [review]
Just one more thing I realized: What was the rationale to rename from "Holder" to "Citizen Name"? Knowing that not everyone has an ENS name, "Holder" would be more appropriate.

**Comment by aahna-ashina** [comment]
> Just one more thing I realized: What was the rationale to rename from "Holder" to "Citizen Name"? Knowing that not everyone has an ENS name, "Holder" would be more appropriate.

@luisivan It was because I saw it in the [mockup](https://github.com/nation3/brand/blob/main/passport/pass/mockup.png) design.  But reverted the change back to "Passport Holder" after @xPi2's code review: https://github.com/nation3/mobile-passport/pull/32/commits/ac3a21548d01ecdf40fb4c0091f15c9db228428d

Screen Shot 2022-06-29 at 5 05 18 PM

Screen Shot 2022-06-29 at 5 05 30 PM

**Pull request merged**